docs/beyond_basics/101-170-dataladrun.rst

```diff
@@ -0,0 +1,474 @@
+In this concrete example, the subdatasets ``fmriprep`` and ``freesurfer`` will each have more than 1000 branches that hold individual job results.
+The only thing left to do now is merging all of these branches into :term:`master` -- and potentially solve any merge conflicts that arise.
+Usually, merging branches is done using the ``git merge`` command with a branch specification.
+For example, in order to merge one job branch into the :term:`master` :term:`branch`, one would need to be on ``master`` and run ``git merge <job branch name>``.
+Given that the subdatasets each contain >1000 branches, and that each ``merge`` would lead to a commit, in order to not inflate the history of the dataset with hundreds of merge commits, two `Octopus merges <https://git-scm.com/docs/git-merge#Documentation/git-merge.txt-octopus>`_ were done - one in each subdataset (``fmriprep`` and ``freesurfer``).
+
+.. findoutmore:: What is an octopus merge?
+
+   Usually a commit that arises from a merge has two *parent* commits: The *first parent* is the branch the merge is being performed from, in the example above, ``master``. The *second parent* is the branch that was merged into the first.
+
+
+   However, ``git merge`` is capable of merging more than two branches simultaneously if more than a single branch name is given to the command.
+   The resulting merge commit has as many parent as were involved in the merge.
+   If a commit has more than two parents, if is affectionately called an "Octopus" merge.
+
+   Octopus merges require merge-conflict-free situations, and will not be carried out whenever manual resolution of conflicts is needed.
+
+The merge command can be assembled quickly.
+As all result branches were named ``job-<JOBID>``, a complete list of branches is obtained with the following command::
+
+   $ git branch -l | grep 'job-' | tr -d ' '
+
+This command line call translates to: "list all branches, of all branches, show me those that contain ``job-``, and remove (``tr -d``) all whitespace.
+This can be given to ``git merge`` as in
+
+.. code-block:: bash
+
+   $ git merge -m "Merge results from job cluster XY" $(git branch -l | grep 'job-' | tr -d ' ')
+
+**Merging with merge conflicts**
+
+When attempting an octopus merge like the one above and a merge conflict arises, the merge is aborted automatically. This is what it looks like::
+
+   $ git merge -m "Merge results from job cluster 107890" $(git branch -l | grep 'job-' | tr -d ' ')
+    Fast-forwarding to: job-107890.0
+    Trying simple merge with job-107890.1
+    Simple merge did not work, trying automatic merge.
+    ERROR: logs/CITATION.md: Not merging symbolic link changes.
+    fatal: merge program failed
+    Automated merge did not work.
+    Should not be doing an octopus.
+    Merge with strategy octopus failed.
+
+This merge conflict arose in the ``fmriprep`` subdataset an originated from the fact that each job generated a ``CITATION.md`` file with minimal individual changes.
```

Comment:
Personally, I would mention that octopus cannot have conflicts, but that is also often no problem for per-sample processing.

That it doesn't work out of the box with fmriprep is a footnote in this context, and I would move the mention and explanation entirely into a findoutmore box.